lib/pages/edit_profile_page.dart

```diff
@@ -268,10 +303,11 @@ class  EditProfileFormState  extends State<EditProfileForm> {
       }
     }catch (e)
     {
+      throw Exception('Error updating profile: $e');
```

Comment:
Im just wondering, should we let the user know that there was an error when they tried to update their profile? Let me know what you think?

Reply:
I think we should.

Reply:
I honestly don't know. Some users might like to see it instead of wondering why it didn't work?